core/src/main/java/cf/dmms/app/core/author/AuthorDto.java

```diff
@@ -0,0 +1,35 @@
+package cf.dmms.app.core.author;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import javax.validation.constraints.NotBlank;
+
+public class AuthorDto {
+
+	private Long id;
+
+	@NotBlank
+	private String name;
+
+	@NotBlank
+	private String surname;
+
+	@JsonCreator
+	public AuthorDto(@JsonProperty("name") String name, @JsonProperty("surname") String surname) {
```

Comment:
Jeśli mamy pole na id, ale brak możliwości jego ustawienia, to jak mamy to przekazać na front? 
Bo przy rejestracji faktycznie nie jest potrzebne.